app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think here FrameLayout parent viewGroup not necessary

Reply:
done